planners/at_bfs_f-ffparser/src/ipc2014_rwa.hxx

```diff
@@ -29,26 +29,26 @@ namespace aptk {
 namespace search {
 
 namespace ipc2014 {
-	
+
 	template <typename State>
 	class Node {
 	public:
 		typedef Fibonacci_Open_List< Node >	Open_List;
-		typedef	State				State_Type;	
+		typedef	State				State_Type;
+
+		typedef Node<State>* Node_Ptr;
+		typedef typename std::list< Node<State>* >                     	Node_Ptr_List;
+		typedef typename std::list< Node<State>* >::reverse_iterator    	Node_Ptr_List_Rit;
+		typedef typename std::list< Node<State>* >::iterator            	Node_Ptr_List_It;
```

Comment:
Hello again @noskill,

thanks for this Anatoly. This code was put together in a rather haphazard way to meet the IPC 2014 competition deadline, and it shows.

Miquel.